addons/mail/models/mail_thread_blacklist.py

```diff
@@ -47,7 +48,18 @@ class MailBlackListMixin(models.AbstractModel):
     def _compute_email_normalized(self):
         self._assert_primary_email()
         for record in self:
-            record.email_normalized = tools.email_normalize(record[self._primary_email])
+            if not record[self._primary_email]:
```

Comment:
This cannot be accepted as this is a huge change. For example if I do on a template: ``email_to: {{ "object.name" <object.email_normalized> }}`` this new value of email_normalized breaks it. Normalized email is an email you should be able to use in a formataddr, as it is, ... in short: an email you should consider complete (it can be wrong or buggy, but does not contain any formatting).

Reply:
I have https://github.com/odoo/odoo/pull/74474 ongoing that handles a big better "multi emails" (coming from an old feedback in 12, recently rebased on 13). Maybe it could partially solve some issues raised here ...